### ISSUE TITLE
Check-out of AppData file

### ISSUE THREAD

**Issue by ishani** [botman99/P4SimpleScc#1]
First up, thank you for writing this extension, it's exactly what I've wanted ever since experiencing the official P4 one!

Been working out great so far, the only quirk I've found is a message to check-out an uncontrolled JetBrains settings file on startup (presumably Resharper tools doing things in the background). 

![image](https://user-images.githubusercontent.com/142049/172829706-da9faf87-1307-4e1e-bbaa-8087da4ca279.png)

This was using the [v1.4.0](https://github.com/botman99/P4SimpleScc/tree/v1.4.0) version released May 21, Visual Studio 2019

**Comment by botman99** [comment]
You can create a [.p4ignore](https://www.perforce.com/manuals/cmdref/Content/CmdRef/P4IGNORE.html) file at the root of your workspace and add files to that that you don't wish to be checked into source control.  You'll have to set the P4IGNORE environment setting or use "p4 set" to set it to the name you pick for the .p4ignore file (like 'p4ignore.txt', etc).

**Comment by ishani** [comment]
Oh we do use ignores - but note this file is deep out in my %APPDATA% directory, a long way from the solution / workspace root, hence my confusion.

**Comment by ishani** [comment]
(I'm curious if this is a file operation from VS (via one of the JetBrain extensions) that P4SimpleScc is catching and associating with the open solution for some reason.)

**Comment by botman99** [comment]
Perforce should not be checking out any files that aren't in your workspace.  If you try to manually check out that file, Perforce should complain that it is not in the client view.  I'm not sure how that file is being checked out or added to source control.

**Comment by botman99** [comment]
If you click 'Yes' to check it out, do you get an error from the Perforce server?  I'm guessing there's some association in Visual Studio between that file and some other file where VS thinks that if you modify one, you may also modify the other.

**Comment by botman99** [comment]
I can probably modify P4SimpleScc to check the file path of files trying to be checked out and only allow files that are within the client workspace to prompt for checkout.  That should fix this.

Thanks for pointing this out, by the way.  :)

**Comment by ishani** [comment]
here's the log tail (mildly censored)

```
06/09/2022 23:58:18 - Connection to server successful.
06/09/2022 23:59:52 - command: p4.exe -p xxxxxx.com:1666 -u user -c my_workspace fstat -T "action" "C:\Users\User\AppData\Roaming\JetBrains\Shared\vAny\GlobalSettingsStorage.DotSettings"
response: 
error: Path 'C:\Users\User\AppData\Roaming\JetBrains\Shared\vAny\GlobalSettingsStorage.DotSettings' is not under client's root 'D:\TheRoot\OfMyWorkspace'.
```

The request pops as the ReSharper toolkit extension is async loaded.

**Comment by botman99** [comment]
This should now be fixed in the 1.5 release with [this](https://github.com/botman99/P4SimpleScc/commit/c6be3e3a0b1144af442d57e73837650384866e7d) commit.

If you have verbose output enabled, you will still see an error message about the file being outside the client's root folder, but there should not be a confirmation dialog appearing for that file anymore.  The error message is what the P4SimpleScc extension uses to determine that the file can't be checked out.  It will look something like this:

error: Path 'C:\Users\xxx\AppData\Roaming\Grepy2\Grepy2.ini' is not under client's root 'D:\Projects\Junk'.
